### ISSUE TITLE
Update lengend-state for node 18.15.0

### ISSUE THREAD

**Issue by krausest** [krausest/js-framework-benchmark#1211]
Whilst updating to Node 18.15.0 I tried to bump versions for legend-state, but I couldn't make it work.
After reverting all changes and setting `NODE_OPTIONS=--openssl-legacy-provider` the implementation is still incorrect:

```
Executing frameworks/non-keyed/legend-state and benchmark 03_update10th1k_x16 failed: checkElementContainsText pierce/tbody>tr:nth-of-type(991)>td:nth-of-type(2)>a failed. expected  !!!, but was cheap pink house
Executing frameworks/non-keyed/legend-state and benchmark 04_select1k failed: checkElementContainsText pierce/tbody>tr:nth-of-type(1000)>td:nth-of-type(1) failed. expected 1000, but was 2000
Executing frameworks/non-keyed/legend-state and benchmark 23_update5-memory failed: checkElementContainsText pierce/tbody>tr:nth-of-type(1)>td:nth-of-type(2)>a failed. expected  !!!, but was short white cookie
```
I moved legend-state into the broken-frameworks folder.

@jmeistrich Can you take a look at it please?

**Comment by jmeistrich** [comment]
Yes no worries, I'm just about to release version 1.0 so I was planning to update it here anyway :).

**Comment by krausest** [comment]
fixed in PR #1269
